cookbooks/aws-parallelcluster-install/files/default/ec2_udev_rules/ec2_dev_2_volid.py

```diff
@@ -24,11 +25,27 @@ def get_imdsv2_token():
     return headers
 
 
+def validate_device_name(device_name):
+    """
+    Validate an argument used to build a subprocess command against a regex pattern.
+
+    The validation is done after forcing the encoding to be the standard Python Unicode / UTF-8
+    :param device_name: an argument string to validate
+    :raise: Exception if the argument fails to match the patter
+    :return: True if the argument matches the pattern
+    """
+    match = re.match(r"^(\w){2,}$", device_name)
```

Comment:
Why` {2,}`?

Reply:
The block device names all have at least 2 alphanumeric chars [persistent_block_device_naming](https://wiki.archlinux.org/title/persistent_block_device_naming). For security purposes though, this regex constraint isn't really a hard requirement though. I can remove it.

Reply:
Since the goal of this change is security enforcement I would avoid introducing formatting constraints that may be out of scope and introduce risk (e.g. what if the name format changes in future? not highly probably I'd say but still can be avoided)